src/Common/common.cpp

```diff
@@ -0,0 +1,5 @@
+#include "common.h"
```

Comment:
Please remove Common class with files. Probably you will not have a class called like this in the project. You will create another files here later when you need it.

Reply:
I removed Common.h and Common.cpp from project